templates/documentation/sdks/api-clients/README.md

```diff
@@ -15,26 +15,26 @@ Seamlessly integrate video on demand or live streaming into your current tech st
 
 <div class="hagrid">
 
-{% include "_partials/hagrid-item.md" title: "NodeJS", image: "https://s3.eu-central-1.amazonaws.com/api.video-strapi/node_031ab5d518.svg", subtitle: "API client",  link: "././apivideo-nodejs-client.md" %}
+{% include "_partials/hagrid-item.md" title: "NodeJS", image: "/_assets/icons/sdk_icons/NodeJs.svg", subtitle: "API client",  link: "././apivideo-nodejs-client.md" %}
 
-{% include "_partials/hagrid-item.md" title: "Python", image: "https://s3.eu-central-1.amazonaws.com/api.video-strapi/python_96b8a9b719.svg", subtitle: "API client",  link: "././apivideo-python-client.md" %}
+{% include "_partials/hagrid-item.md" title: "Python", image: "/_assets/icons/sdk_icons/Python.svg", subtitle: "API client",  link: "././apivideo-python-client.md" %}
 
-{% include "_partials/hagrid-item.md" title: "PHP", image: "https://s3.eu-central-1.amazonaws.com/api.video-strapi/php_459780cd53.svg", subtitle: "API client",  link: "././apivideo-php-client.md" %}
+{% include "_partials/hagrid-item.md" title: "PHP", image: "/_assets/icons/sdk_icons/php.svg", subtitle: "API client",  link: "././apivideo-php-client.md" %}
 
-{% include "_partials/hagrid-item.md" title: "Go", image: "https://s3.eu-central-1.amazonaws.com/api.video-strapi/go_3b0d7ddcbc.svg", subtitle: "API client",  link: "././apivideo-go-client.md" %}
+{% include "_partials/hagrid-item.md" title: "Go", image: "/_assets/icons/sdk_icons/Go.svg", subtitle: "API client",  link: "././apivideo-go-client.md" %}
 
-{% include "_partials/hagrid-item.md" title: "C#", image: "https://s3.eu-central-1.amazonaws.com/api.video-strapi/c_sharp_55a8990e3a.svg", subtitle: "API client",  link: "././apivideo-csharp-client.md" %}
+{% include "_partials/hagrid-item.md" title: "C#", image: "/_assets/icons/sdk_icons/C.svg", subtitle: "API client",  link: "././apivideo-csharp-client.md" %}
```

Comment:
In case we support `C` language one day maybe name it `Csharp.svg` would be more appropriate

Reply:
That is one very precise comment 😁  Making the change 👍